src/diagnostics.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * this module contains functions to show jest test results in
+ * vscode inspector via the DiagnosticsCollection.
+ */
+import * as vscode from 'vscode'
+// import { DiagnosticCollection, Uri, Diagnostic, Range, DiagnosticSeverity } from 'vscode'
+import { TestFileAssertionStatus } from 'jest-editor-support'
+import { TestReconciliationState } from './TestReconciliationState'
+
+export function updateDiagnositics(testResults: TestFileAssertionStatus[], diagnostics: vscode.DiagnosticCollection) {
```

Comment:
`updateDiagnositics`, typo: `updateDiagnostics`

Reply:
Oops, indeed... Looks like you already merged it, I will submit a new PR then